cli/client/generic.go

```diff
@@ -133,6 +133,21 @@ func (client *RestClient) Post(path string, obj interface{}) error {
 	return nil
 }
 
+// PostBody sends a POST request with obj as the payload to the given path
+// and additionally returns the response body
+func (client *RestClient) PostBody(path string, obj interface{}) ([]byte, error) {
```

Comment:
Why not return the entire response, that way headers can be inspected as well :)